### PR TITLE
Metadata: display_name syntax for algorithm title

### PR DESCRIPTION
**Pull request by rldhont**
Defined a new metadata line to set the algorithm title without changes the algorithm id.

### CODE REVIEW

**Comment by JanCaha** [comment]
Nice 👍 This should be quite useful.

Would you mind adding a new test, so that this functionality can be properly checked? Should be as simple as adding new function into **test_algorithm.py** (there are plenty examples there) and you can add new testing script into **qgis-processing-r/processing_r/test/data/**.

File: website/pages/script-syntax.md [hunk 1]
```diff
@@ -6,7 +6,14 @@ The script loads necessary packages by itself. Combination of _sp_ and _rgdal_ o
 
 ## Metadata
 
+The R script's file name is used to defined the script name and id in the R processing provider. You can ovveride these
```
Comment (Gustry) [inline website/pages/script-syntax.md:9]:
```suggestion
The R script's file name is used to define the script name and id in the R processing provider. You can override these
```

File: website/pages/script-syntax.md [hunk 2]
```diff
@@ -6,7 +6,14 @@ The script loads necessary packages by itself. Combination of _sp_ and _rgdal_ o
...
+default metadata with these lines.
+
 `##script_name=name` _script_name_ is the name of the script; under this name, it will be listed in processing toolbox.
+It is also used to defined the script id in the R processing provider.
```
Comment (Gustry) [inline website/pages/script-syntax.md:13]:
```suggestion
It is also used to define the script id in the R processing provider.
```

File: website/pages/script-syntax.md [hunk 3]
```diff
@@ -6,7 +6,14 @@ The script loads necessary packages by itself. Combination of _sp_ and _rgdal_ o
...
+
+`##script_title=display_name` _script_title_ is the title of the script; under this title, it will be listed in processing toolbox.
+It overried the _name_ if it is defined after.
```
Comment (Gustry) [inline website/pages/script-syntax.md:16]:
```suggestion
It overrides the _name_ if it is defined after.
```

**Comment by rldhont** [comment]
Thanks @Gustry 

@JanCaha I have added tests. Ready for review

**Comment by JanCaha** [comment]
Great, thank you!

**Pull request merged**